<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>500</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>扫雷</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/res.qrc">
    <normaloff>:/png/bomb</normaloff>:/png/bomb</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLCDNumber" name="timeKeeper">
      <property name="smallDecimalPoint">
       <bool>false</bool>
      </property>
      <property name="digitCount">
       <number>8</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="startButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>4</width>
        <height>4</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="iconSize">
       <size>
        <width>48</width>
        <height>48</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
      <property name="autoRepeatInterval">
       <number>100</number>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLCDNumber" name="score">
      <property name="smallDecimalPoint">
       <bool>false</bool>
      </property>
      <property name="digitCount">
       <number>8</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
      <property name="intValue" stdset="0">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QGraphicsView" name="sceneView">
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>240</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>2400</width>
        <height>2400</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>12</width>
        <height>12</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>240</width>
        <height>240</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="settings">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="about"/>
   </widget>
   <widget class="QMenu" name="game">
    <property name="title">
     <string>游戏</string>
    </property>
    <addaction name="newGame"/>
    <addaction name="restartGame"/>
    <addaction name="exitGame"/>
   </widget>
   <addaction name="game"/>
   <addaction name="settings"/>
  </widget>
  <action name="exitGame">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="newGame">
   <property name="text">
    <string>新游戏</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="restartGame">
   <property name="text">
    <string>重新开始</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="about">
   <property name="text">
    <string>关于</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res/res.qrc"/>
 </resources>
 <connections/>
</ui>
